src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -499,29 +537,6 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 GetEmitter()->emitIns_R_R_R(ins, emitSize, targetReg, op1Reg, op2Reg, opt);
                 break;
 
-            case NI_AdvSimd_CompareEqual:
```

Comment:
This part was removed in favor of the table driven approach as it was simpler to handle the instructions explicitly rather than the intrinsics.